src/dlls/mscoree/coreclr/CMakeLists.txt

```diff
@@ -120,13 +120,18 @@ else()
     list(APPEND CORECLR_LIBRARIES
         ${START_WHOLE_ARCHIVE} # force all PAL objects to be included so all exports are available
         coreclrpal
-        tracepointprovider
         ${END_WHOLE_ARCHIVE}
         mscorrc_debug
         palrt
     )
 endif(WIN32)
 
+if(CMAKE_SYSTEM_NAME STREQUAL Linux)
```

Comment:
This is related to the comment in `src/pal/src/CMakeLists.txt` to avoid the warning about empty symbols. There are some other objects with no symbols for macOS, but the warnings are visible in master logs as well. I will try to fix those warnings in a separate PR.